src/pages/claim/index.vue

```diff
@@ -200,11 +201,13 @@ function formatRewardsData(data?: BalanceMap): ProtocolRewardRow[] {
  * @summary Fetches bb-a-USD rate as an appoximation of USD price.
  */
 async function getBBaUSDPrice() {
-  if (isMainnet.value) {
-    const appoxPrice = bnum(await bbAUSDToken.getRate()).toNumber();
+  if (TOKENS.Addresses.bbaUSD) {
+    const approxPrice = bnum(await bbAUSDToken.getRate()).toNumber();
```

Comment:
Yeah this bbAUSDToken.getRate() should be for whatever the current reward token is, I imagine the latest version of the bb-a-USD pool.

Reply:
Fixed